### PR TITLE
Probe::isDisabled -->> Probe::enabled

### PR DESCRIPTION
**Pull request by klshrinidhi**
Addressing #1471.

### CODE REVIEW

File: OpenSim/Simulation/Model/Probe.h [hunk 1]
```diff
@@ -123,8 +123,8 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Probe, ModelComponent);
 //==============================================================================
 // PROPERTIES
 //==============================================================================
-    /** Enabled by default. **/
-    OpenSim_DECLARE_PROPERTY(isDisabled, bool,
+    /** Enabled (true) by default. **/
+    OpenSim_DECLARE_PROPERTY(enabled, bool,
```
Comment (tkuchida) [inline OpenSim/Simulation/Model/Probe.h]: Comment on L128 should be revised to "Flag indicating whether the Probe is enabled"

File: OpenSim/Simulation/Model/Probe.h [hunk 2]
```diff
@@ -160,10 +160,12 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Probe, ModelComponent);
     /** Get the number of states in the underlying SimTK::Measure. */
     int getNumInternalMeasureStates() const;
 
-    /** Returns true if the Probe is disabled or false if the probe is enabled. */
-    bool isDisabled() const;
-    /** %Set the Probe as disabled (true) or enabled (false). */
-    void setDisabled(bool isDisabled);
+    /** Returns true if the Probe is enaabled and false if the probe is
```
Comment (tkuchida) [inline OpenSim/Simulation/Model/Probe.h]: enaabled -> enabled

File: OpenSim/Simulation/Model/Probe.h (hunk 1 above)
Comment (klshrinidhi) [inline OpenSim/Simulation/Model/Probe.h]: Done.

File: OpenSim/Simulation/Model/Probe.h (hunk 2 above)
Comment (klshrinidhi) [inline OpenSim/Simulation/Model/Probe.h]: Done.

**Comment by tkuchida** [comment]
:+1: All concerns addressed by 4e672bd

**Comment by aymanhab** [comment]
LGTM. Thanks @klshrinidhi Merging.

**Pull request merged**